src/settings/components/Sets/SetsForm/SetsForm.js

```diff
@@ -119,5 +119,6 @@ SetsForm.defaultProps = {
 };
 
 export default stripesFinalForm({
+  keepDirtyOnReinitialize: true,
```

Comment:
Could you please clarify the use case for this?

Reply:
We submit data to BE, but we don't need reset them (form values). In accordance with a fact that we will have BE side validation for unique values for several field. We can get errors from BE and we will need continue editing our form values.